package.json

```diff
@@ -16,80 +15,82 @@
     "e2e": "npx playwright test",
     "lint": "nx run-many --all --target=lint",
     "format": "prettier --write \"{apps,libs}/**/*.{ts,md,html}\" \"!{apps,libs}/**/coverage\" ",
-    "format:check": "prettier --list-different \"{apps,libs}/**/*.{ts,md}\" \"!{apps,libs}/**/coverage\""
+    "format:check": "prettier --list-different \"{apps,libs}/**/*.{ts,md}\" \"!{apps,libs}/**/coverage\"",
+    "check-update": "ng update && ncu -u",
```

Comment:
Can we remove these 2 lines? It's an opinionated pattern not strictly needed for the update